base-notebook/test/test_package_managers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import logging
+import pytest
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "package_manager, cmd",
+    [
+        ("apt", "--version"),
+        ("conda", "--version"),
+        ("mamba", "--version"),
+        ("npm", "--version"),
```

Comment:
`pip` should be also included, I suppose.

Reply:
Yes off course 👍

Reply:
Done